src/constants.ts

```diff
@@ -22,5 +22,6 @@ export const VOYAGER_GOERLI_CONTRACT_API_URL = "https://goerli.voyager.online/ap
 export const VOYAGER_MAINNET_CONTRACT_API_URL = "https://voyager.online/api/contract/";
 
 export const SHORT_STRING_MAX_CHARACTERS = 31;
+export const OPENZEPPELIN_EXECUTE_FUNCTION = "execute";
```

Comment:
Better to have it as a `static readonly` string on OpenZeppelinAccount clas itself.